frontend/src/features/leases/detail/LeaseContainer.tsx

```diff
@@ -55,15 +64,19 @@ export const LeaseContainer: React.FunctionComponent<ILeaseAndLicenseContainerPr
     throw Error('The requested lease page does not exist');
   }
   return (
-    <LeaseLayout>
-      <LeaseBreadCrumb
-        leaseId={props.match.leaseId}
-        leasePage={leasePage}
-        onClickManagement={onClickManagement}
-      />
-      <BackToSearchButton />
-      <LeaseIndex currentPageName={leasePageName} leaseId={lease?.id}></LeaseIndex>
-    </LeaseLayout>
+    <>
+      <LeaseLayout>
+        <LeaseBreadCrumb
+          leaseId={props.match.leaseId}
+          leasePage={leasePage}
+          onClickManagement={onClickManagement}
+        />
+        <LeaseHeader lease={lease} />
+        <BackToSearchButton />
+        <LeaseIndex currentPageName={leasePageName} leaseId={lease?.id}></LeaseIndex>
```

Comment:
I see 3 ways to get the lease Id in this section:

- props?.match?.params?.leaseId
- props.match.leaseId
- lease?.id

Any change we could just pick one? I understand **lease** is undefined until we find it by the route param but once we find it we not use that instead of the route param?

Reply:
Will correct in follow up PR, thanks.